site/content/en/docs/reference/kubectl-kueue/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Kubectl Kueue Plugin"
+date: 2024-05-06
+weight: 10
+description: >
+  The kubectl-kueue plugin, kueuectl, allows you to manage Kueue resources.
+---
```

Comment:
shouldn't this have the contents that are currently on kubectl-kueue.md?

Reply:
Fixed